packages/eslint-plugin/tests/rules/consistent-type-definitions.test.ts

```diff
@@ -347,6 +359,61 @@ export declare interface Test {
 export declare type Test = {
   foo: string;
   bar: string;
+}
+      `,
+    },
+    {
+      code: noFormat`
+type Foo = ({
+  a: string;
+});
+      `,
+      errors: [
+        {
+          line: 2,
+          messageId: 'interfaceOverType',
+        },
+      ],
+      output: `
+interface Foo {
+  a: string;
+}
+      `,
+    },
+    {
+      code: noFormat`
+type Foo = ((((((((({
+  a: string;
+})))))))));
+      `,
```

Comment:
Could you add a test case like

```ts
type Foo = ((({
  a: string;
})))

// next statement has a semi
const bar = 1;
```

To ensure we are handling the no-semi case?

Reply:
sure! added 👍

Reply:
Note that the risk of messing up a semi/no-semi issue is also lessened now that the fixer isn't querying for it; it's just deleting up to `node.range[1]`.